client/src/components/geneExpression/expressionButtons.js

```diff
@@ -60,27 +60,37 @@ class Expression extends React.Component {
         <CellSetButton {...this.props} eitherCellSetOneOrTwo={1} />
         <CellSetButton {...this.props} eitherCellSetOneOrTwo={2} />
         {!differential.diffExp ? (
-          <Button
-            style={{ marginTop: 10 }}
-            disabled={!haveBothCellSets}
-            intent="primary"
-            fill
-            type="button"
-            onClick={this.computeDiffExp.bind(this)}
+          <Tooltip
+            content="Add two cells selections, see the top 15 differentially expressed genes between them"
```

Comment:
What does "add two cells selections" mean?    Add as in summation?

Perhaps:  "Compute differential expression between cell selection one and two."